2010-04-20  Michael Tautschnig  <mt@debian.org>

	* setup-storage/Volumes.pm: Use VG size for volume group size, not space in
		use. Thanks Mathieu Alorent for reporting this.
Index: trunk/lib/setup-storage/Volumes.pm
===================================================================
--- trunk.orig/lib/setup-storage/Volumes.pm
+++ trunk/lib/setup-storage/Volumes.pm	
@@ -377,7 +377,7 @@
     my %vg_info = get_volume_group_information($vg);
     if (%vg_info) {
       $FAI::current_lvm_config{$vg}{size} = &FAI::convert_unit(
-        $vg_info{alloc_pe_size} . $vg_info{alloc_pe_size_unit});
+        $vg_info{vg_size} . $vg_info{vg_size_unit});
     } else {
       $FAI::current_lvm_config{$vg}{size} = "0";
     }
